melodic/distribution.yaml

```diff
@@ -3806,6 +3806,26 @@ repositories:
       url: https://github.com/ros-gbp/libg2o-release.git
       version: 2018.3.25-0
     status: maintained
+  librealsense:
+    doc:
+      type: git
+      url: https://github.com/IntelRealSense/librealsense.git
+      version: legacy
+    source:
+      type: git
+      url: https://github.com/IntelRealSense/librealsense.git
+      version: legacy
+    status: maintained
+  librealsense2:
+    doc:
+      type: git
+      url: https://github.com/IntelRealSense/realsense-ros.git
+      version: development
+    source:
+      type: git
+      url: https://github.com/IntelRealSense/realsense-ros.git
+      version: 2.2.9    
```

Comment:
This is failing CI because the source entry can't point to a tag, it must point to branch.

Reply:
This is still failing CI because of this problem.

Reply:
I just changed it to point to the development branch.